SPECS/prometheus-adapter/prometheus-adapter.spec

```
@@ -1,18 +1,24 @@
 Summary:        Kubernetes Custom, Resource, and External Metric APIs implemented to work with Prometheus.
 Name:           prometheus-adapter
 Version:        0.10.0
-Release:        6%{?dist}
+Release:        7%{?dist}
 License:        Apache-2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/kubernetes-sigs/prometheus-adapter
 Source0:        https://github.com/kubernetes-sigs/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildRequires:  golang
-BuildRequires:  prometheus
 
 %description
 Implementation of Prometheus via Kubernetes Custom, Resource, and External Metric API.
 
+%package docs
+Summary:        prometheus-adapter docs
+Requires:       %{name}
```

Can we make these requirements more strict with respect to versioning? We want users to have their docs match the version they are using.

```suggestion
Requires:       %{name} = %{version}-%{release}
```